### ISSUE TITLE
#-488323171------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1855]
<strong>PageName:</strong> FeedDetailsPage  

<strong>Source:</strong>`<div id="attr-label-units" class="sc-dUWDJJ eBsayS" aria-describedby="calcite-tooltip-6519c4b5-19ce-7b1a-e91c-13ee9c0085d2">units</div>`

<strong>Element Location: </strong>#attr-label-units

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\FeedDetailsPage\-488323171_All page content should be contained by landmarks.png`
